components/KiboCollectedProduct.vue

```diff
@@ -0,0 +1,213 @@
+        />
+      </div>
+      <div class="kibo-collectedProduct__quantitySelector-wrapper">
+        <p>Qty:</p>
+        <SfQuantitySelector
+          :qty="quantity"
+          :min="1"
+          :max="10"
+          @input="handleQuantitySelectorInput"
+        />
+      </div>
+      <div class="kibo-collectedProduct__details-wrapper">
+        <SfAccordion v-if="options.length" open="Details" show-chevron>
+          <SfAccordionItem header="Details">
+            <div v-for="(option, index) in options" :key="index">
+              <div class="sf-property">
+                <span class="sf-property__name">{{ option.name }}</span>
+                <span class="sf-property__value"> {{ option.value }}</span>
+              </div>
+            </div>
+          </SfAccordionItem>
+        </SfAccordion>
+      </div>
+    </div>
+    <div class="kibo-collectedProduct__verticalDivider"></div>
+    <div class="kibo-collectedProduct__fullfilment">
+      <KiboFulfillmentOptions
+        :fulfillment-options="supportedFulfillmentTypes"
+        @click="handleStoreLocatorClick"
+      />
+    </div>
+    <div class="kibo-collectedProduct__remove">
+      <SfButton class="kibo-collectedProduct__remove-button" @click="handleRemoveCartItem">
+        <font-awesome-icon :icon="['fas', 'trash-alt']" />
+      </SfButton>
+    </div>
+  </div>
+</template>
+<script>
+import {
+  SfPrice,
+  SfIcon,
+  SfImage,
+  SfCircleIcon,
+  SfButton,
+  SfQuantitySelector,
+  SfLink,
+  SfProperty,
+  SfAccordion,
+  SfRadio,
+} from "@storefront-ui/vue"
+import KiboFulfillmentOptions from "@/components/KiboFulfillmentOptions.vue"
+import { useCart, useUiState } from "@/composables"
+
+export default defineComponent({
+  name: "KiboCollectedProduct",
+  components: {
+    SfButton,
+    SfIcon,
+    SfImage,
+    SfCircleIcon,
+    SfPrice,
+    SfQuantitySelector,
+    SfLink,
+    SfProperty,
+    SfAccordion,
+    SfRadio,
+    KiboFulfillmentOptions,
+  },
+  model: {
+    prop: "qty",
+  },
+  props: {
+    /**
+     * Product image
+     * It should be an url of the product
+     */
+    image: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Product image width, without unit
+     */
+    imageWidth: {
+      type: [String, Number],
+      default: 140,
+    },
+    /**
+     * Product image height, without unit
+     */
+    imageHeight: {
+      type: [String, Number],
+      default: 200,
+    },
+    /**
+     * Product title
+     */
+    title: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Product regular price
+     */
+    regularPrice: {
+      type: [Number, String],
+      default: null,
+    },
+    /**
+     * Product special price
+     */
+    specialPrice: {
+      type: [Number, String],
+      default: null,
+    },
+    /**
+     * Selected quantity
+     */
+    qty: {
+      type: [Number, String],
+      default: 1,
+    },
+    /**
+     * Link to product
```

Comment:
give single line comments instead of two like below

Reply:
Done